airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -90,8 +90,17 @@ def get_link(
         *,
         ti_key: TaskInstanceKey,
     ):
-        job_id = XCom.get_value(key="job_id", ti_key=ti_key)
-        return BIGQUERY_JOB_DETAILS_LINK_FMT.format(job_id=job_id) if job_id else ""
+        job_id_params = XCom.get_value(key="job_id_params", ti_key=ti_key)
```

Comment:
Why are we using `job_id_params` instead of the previous `job_id`? Is this not a breaking change? I think fixes should not break the provider except we are bumping to a major.

Reply:
This has been changed following the suggestion in this comment: https://github.com/apache/airflow/pull/31457#discussion_r1228601415

Reply:
We should probably do the renaming when we want to release a major version because it's a breaking change for anyone using the `job_id` already. Add #TODO and explain to bump this in a major release

Reply:
@ephraimbuddy the change is only in the link class and currently the link doesnt work. I don't think its a breaking change. I can't think of any user code that will be broken by this change.

Reply:
But the REST API broke?